src/components/NewGalaxyModal.js

```diff
@@ -7,29 +7,40 @@ import { withModalDrawer } from 'src/components/ModalDrawer'
 import TitleBar from 'src/components/TitleBar'
 import { machineTypes } from 'src/data/machines'
 import { Ajax } from 'src/libs/ajax'
+import { currentApp } from 'src/libs/cluster-utils'
 import colors from 'src/libs/colors'
 import * as Utils from 'src/libs/utils'
 
 
 const styles = {
   whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
-  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' }
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: '0 1 0%', padding: '1.5rem' }
```

Comment:
Flex-grow 0 is to fix the buttons being all the way at the bottom of the page (per the mocks). If that's wrong, I can definitely revert this @ehanna4

Reply:
the change looks good in the UI to me